hdfs-executor/src/main/java/org/apache/mesos/hdfs/executor/AbstractExecutor.java

```diff
@@ -139,15 +316,15 @@ private void createSymbolicLink() {
       String msg = "Error creating the symbolic link to hdfs binary";
       shutdownExecutor(1, msg, e);
     }
-  }
+  }*/
 
   /**
    * Add hdfs binary to the PATH environment variable by linking it to /usr/bin/hadoop. This
    * requires that /usr/bin/ is on the Mesos slave PATH, which is defined as part of the standard
    * Mesos slave packaging.
    */
```

Comment:
The code to distribute the binaries and place them in the path is commented out so it seems this functionality is missing.